### PR TITLE
`Development`: Document suspicious behavior dashboard

### PR DESCRIPTION
**Pull request by tobias-lippert**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://ls1intum.github.io/Artemis/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://ls1intum.github.io/Artemis/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context

The suspicious behavior dashboard lacks documentation.

### Description

added the documentation and a screenshot to the exam session monitoring on the exam mode feature page.

### Steps for Testing
Code Review
Look at the docs and the screenshot of the feature page.

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### Screenshots

Docs

https://github.com/ls1intum/Artemis/assets/84102468/3b20a077-e50b-427c-a26c-13f5c155326f

Feature overview

image

### CODE REVIEW

File: .idea/runConfigurations/Artemis__Server__LocalVC___LocalCI_.xml
Comment (b-fein) [inline .idea/runConfigurations/Artemis__Server__LocalVC___LocalCI_.xml:1]: Accidentally committed?

File: docs/user/exams/instructor/buttons/suspicious_behavior.png
Comment (b-fein) [inline docs/user/exams/instructor/buttons/suspicious_behavior.png:1]: Some of the new inline buttons are pretty big compared to both the surrounding text and similar examples in existing documentation.

image
Size not consistent within new documentation:
image

Is it possible to scale them down a bit?

**Review by b-fein — APPROVED** [review]
Thanks. :+1:

**Review by MaximilianAnzinger — APPROVED** [review]
Explains everything, no open questions for the feature itself.

Just one question: Do we expect instructors to be familiar with CIDR notation or would it make sense to add an example/explanation/link (also in the UI)? I'm personally fine with how it is but maybe this is an interesting QOL improvement?

**Pull request merged**